### ISSUE TITLE
is this config lightweight?

### ISSUE THREAD

**Issue by alavaelian** [worron/awesome-config#64]
is this config more lightweight than having another wm + a bar +eww or a full desktop-like enlightenment?

**Comment by worron** [comment]
>is this config more lightweight than having another wm + a bar +eww 


This config was designed aiming usage comfort mostly, performance optimization was never high priority. So it ended as feature rich system with many fancy widgets, some of them have very rough workers in background. It's hard to compare with some abstract "another config", considering how many features packed in this one. But I think you have a high chance to find something more lightweight with still decent usability. 

In short, I would never promote it as lightweight among wm configs, but believe it's still better than any full desktop environment in terms of resource usage.

**Comment by alavaelian** [comment]
i see i want to create a lightweight linux distro with awesome wm since I thought is like an windows manager with desktop features and does not need a external bar or widget since awesome can make its own
